postgresql10.6-ci/postgresql.conf

```diff
@@ -0,0 +1,643 @@
+# -----------------------------
+# PostgreSQL configuration file
+# -----------------------------
```

Comment:
@Angelfentanez For this you used a psql 10 file or just added the 9.6?

Reply:
@ruiztulio I just added the one found in the 9.6 image, was looking for changes in the configuration file between these versions but found nothing. Should we play it safe and create a new one based on postgres 10 and set the same values as in 9.6?

Reply:
@Angelfentanez yes